pkg/transforms/common.go

```diff
@@ -40,6 +40,16 @@ func commonProperties(resource v1.Object) map[string]interface{} {
 	if resource.GetLabels() != nil {
 		ret["label"] = resource.GetLabels()
 	}
+
+	annotations := resource.GetAnnotations()
+
+	if annotations != nil {
+		// This annotation is large and useless
+		delete(annotations, "kubectl.kubernetes.io/last-applied-configuration")
```

Comment:
Can we make this more generic to prevent any other annotations with too much data? I'm thinking we could set a max of 64 characters in the value. We could either ignore the annotation completely or truncate the value.

Reply:
Querying for annotations with a JSON string in the value will likely cause problems.

Reply:
I ended up excluding annotations with 64 characters.

Reply:
That works for now, we can change it to truncate if we receive complaints about missing annotations.